src/test/java/jenkins/scm/impl/mock/MockSCMController.java

```diff
@@ -80,6 +82,16 @@ public static MockSCMController create() {
         return c;
     }
 
+    public MockSCMController withLatency(@NonNull MockLatency latency) {
+        this.latency = latency;
+        return this;
+    }
+
+    public MockSCMController withoutLatency() {
```

Comment:
Just create a `MockLatency.none()` to cancel. That could also avoid null checks elsewhere.